src/network/requestQueue/socketRequest.spec.js

```diff
@@ -162,32 +156,5 @@ describe('Network > SocketRequest', () => {
         },
       })
     })
-
-    it('emits NETWORK_REQUEST_TIMEOUT', async () => {
```

Comment:
This test has been moved in to `requestQueue/index.spec.js` due to the timeout interval now been set in `index.js.